docs/source/markdown/podman.1.md

```diff
@@ -28,6 +28,10 @@ The CGroup manager to use for container cgroups. Supported values are cgroupfs o
 Note: Setting this flag can cause certain commands to break when called on containers previously created by the other CGroup manager type.
 Note: CGroup manager is not supported in rootless mode when using CGroups Version V1.
 
+**--cgroup-check-user**=*boolean*
```

Comment:
I feel like this should be part of the `--cgroup` flag to `podman run` and `podman create` - we should never be swapping cgroups otherwise.